python/starter_kit/en/START_WELL.md

```diff
@@ -0,0 +1,67 @@
+# Starting well with Python
+
+Programming is a domain very interesting, lucrative and vast.
+Begin with Python is really often a judicious choice for many reasons.
+
+Here are some of them:
+- Python has a very simple syntax, that permits to get compact and lisible codes ;
+- Python is very complete. Concretely, we can do a huge amount of things with it.
+From AI, throught the Web backend, to its utilization in the scientific domains ;
+- The language has a huge community of developers with a lot of resources,
+ and some of its functionalities make learning easy for the beginners.
+(for some reasons hard to explain here, but for instance, the language does not enforce the usage of `;` after each expression).
+
+
+### Some good resources to learn well
+
+> On the Internet, lots of tutorials/courses to learn development exist, except many
+> of them are bad because they teach you bad practices or 
+> obsolete things. - [learndev.info](https://www.learndev.info/en)
+
+Indeed, the resources available on the Internet are not always relevant or up-to-date.
+That is why, the website [learndev.info](https://www.learndev.info/en) offers adapted and relevant courses,
+such as the course of **[ TODO ]**.
+
+
+
+### What not to do when you begin
```

Comment:
I think this formulation is a little weird, maybe this one is better ?
```suggestion
### Errors to not do when you begin
```

Reply:
Indeed, more correct and corresponds better to the original title. Fixed!